node-js/node-lab-exercise-part-1.md

```diff
@@ -54,14 +54,21 @@ Visit the [nvm repository on GitHub](https://github.com/creationix/nvm) to get s
 
 Follow the installation steps in nvm's readme. Most importantly, these steps:
 
-1. Execute the [install script](https://github.com/creationix/nvm#install-script) using cURL.
-   - Pay attention to the end of the install script because it'll say if it installed nvm's startup scripts in your `.bash_profile`, your `.profile`, or your `.bashrc`. You'll need this information if you need to troubleshoot your installation.
-2. [Verify your nvm installation](https://github.com/creationix/nvm#verify-installation) by typing `command -v nvm` in your terminal. You should just see the output `nvm` in your terminal. (See their readme to read about why `which nvm` does not work. Do not be alarmed that `which nvm` returns nothing!)
+1. Install nvm via Homebrew: `brew install nvm`
+2. Create a system directory for nvm: `mkdir ~/.nvm`
+3. Open your `.bash_profile`, `.profile`, `.zshrc` or `.bashrc` file and append the following lines:
```

Comment:
Change to just `.zshrc` since the current iteration of the curriculum assumes zsh only